package.json

```diff
@@ -27,32 +27,22 @@
   },
   "scripts": {
     "build": "node build.js",
+    "lint": "./node_modules/eslint/bin/eslint.js --ext .njk,.js data/ templates/ lib/",
+    "prettylint": "./node_modules/prettylint/bin/cli.js \"data/**/*.{md,scss,js,json,yaml}\" \"lib/**/*.js\"",
```

Comment:
We were using `.eslintignore` and `.prettierignore` to filter out the folders to be linted, so we wouldn't need to use the paths here. I'll take a look into it.